decidim-admin/app/forms/decidim/admin/participatory_space_admin_user_form.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+module Decidim
+  module Admin
+    class ParticipatorySpaceAdminUserForm < ParticipatorySpacePrivateUserForm
+      attribute :role, String
+
+      validates :role, presence: true
+      validates :role, inclusion: { in: ParticipatorySpaceUser::ROLES }
+      def roles
```

Comment:
Can we add a bit of space to improve legibility 🙏🏽 ? 
```suggestion

      def roles
```